dfs/snapshot.go

```diff
@@ -324,9 +325,10 @@ func (dfs *DistributedFilesystem) DeleteSnapshot(snapshotID string) error {
 // DeleteSnapshots deletes all snapshots relating to a particular tenantID
 func (dfs *DistributedFilesystem) DeleteSnapshots(tenantID string) error {
 	// delete the snapshot subvolume
```

Comment:
We should not be calling GetDriver here.

Instead, it should be dfs.GetVolume
and then if you need the driver you can call volume.Driver()